source/reference/beta-program.txt

```diff
@@ -91,6 +91,12 @@ Beta Features
    * - :ref:`c2c-beta-orr`
      - .. include:: /includes/orr-intro.rst
 
+   * - Pre-6.0 Server Version Support 
+     - In {+c2c-beta-program-short+}, you can perform migrations between 
+       clusters running MongoDB server versions 4.2, 4.4, and 5.0. To see 
```

Comment:
@ajayvijayakumar123 is it worthwhile to keep 4.2 in this list?

Reply:
I think so because beta binaries will support 4.2-->4.2. Once we GA pre-6.0 support, I imagine we can remove this section entirely.

Reply:
Ali, sorry for the back and forth. Maria and I just chatted - we think the note should say " ... you can perform migrations between clusters running MongoDB server versions less than 6.0".

Reply:
No problem at all!

Reply:
Updated w/ a tiny wording adjustment (swapped in "older" rather than "less")